python/tests/db/test_distributed_read.py

```diff
@@ -0,0 +1,105 @@
+import copy
```

Comment:
Now the mongomock is removed. We should merge this file into the test_db.py.

Reply:
done